testnets/deardogetestnet/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "deardogetestnet",
+  "assets": [
+    {
+      "description": "The native staking and governance token of the Theta testnet version of the Dear Doge Testnet.",
+      "denom_units": [
+        {
+          "denom": "peaka",
+          "exponent": 0
+        },
+        {
+          "denom": "DORA",
+          "exponent": 18
+        }
+      ],
+      "base": "peaka",
+      "name": "Dear Doge Testnet",
+      "display": "DORA",
+      "symbol": "DORA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/deardogetestnet/images/deardoge.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/deardogetestnet/images/deardoge.png"
```

Comment:
A duplicate image is not needed. Please point this to Doravota.  An image_sync would be best.

Reply:
deardoge's logo may change after that.